src/Elcodi/Component/Sitemap/Profile/SitemapProfile.php

```diff
@@ -17,88 +17,68 @@
 
 namespace Elcodi\Component\Sitemap\Profile;
 
-use Elcodi\Component\Sitemap\Loader\Interfaces\EntityLoaderInterface;
-use Elcodi\Component\Sitemap\Profile\Interfaces\SitemapProfileInterface;
+use Elcodi\Component\Sitemap\Builder\SitemapBuilder;
 
 /**
  * Class SitemapProfile
  */
-class SitemapProfile implements SitemapProfileInterface
+class SitemapProfile
 {
     /**
-     * @var EntityLoaderInterface[]
+     * @var array
      *
-     * Entity loaders
+     * Languages
      */
-    protected $entityLoaders;
+    protected $languages;
 
     /**
-     * @var string
+     * @var SitemapBuilder[]
      *
-     * Path
+     * Array of SitemapBuilders
```

Comment:
Yes, they are. And like this one, a lot, but take in account PHPDocumentor and others ;)